languages/python/exercises/concept/currency-exchange/.docs/hint.md

```diff
@@ -0,0 +1,18 @@
+# General
```

Comment:
The file name can be changed to `hints.md` and using prettify should add blank lines before and after each title.

Reply:
That seems be solved